component.css

```diff
@@ -3,13 +3,18 @@
 @import "./components/navbar/navbar.css";
 @import "./components/sidebar/sidebar.css";
 @import "./components/avatar/avatar.css";
+@import "./components/alert/alert.css";
 @import "./index.css";
 
 :root{
 
     --primary-background:#E2F1EC;
     --primary-color:#005C68;
     --primary-font:'Spartan', sans-serif;
+    --primary-faint-red:rgb(226, 124, 124);;
+    --primary-faint-green:#99e696;
```

Comment:
variable names can be more generalized as the color can be changed. eg: error-color, success-color etc.

Reply:
sure would change it!